classes/metric/concurrent_users_metrics.php

```diff
@@ -0,0 +1,134 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+namespace tool_cloudmetrics\metric;
+
+use cltr_database\collector;
+
+/**
+ * Metric class for new users.
+ *
+ * @package    metric_foobar
+ * @author     Marc-Alexandre Ghaly <marc-alexandreghaly@catalyst-ca.net>
+ * @copyright  2022, Catalyst IT
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+class concurrent_users_metrics extends builtin {
```

Comment:
Isn't this the same as online_users_metric?

Reply:
I understood that online users are users in the sense of right now whereas concurrent users are values from the past, if those are the same I will remove the extra one.

Reply:
In this case the online_users_metric is redundant as concurrent_users_metric can fill both roles.

Reply:
+1, this should just be back filling the online_users_metric

Reply:
I have removed concurrent_users_metric class.